superweather/superweatherUITests/superweatherUITests.m

```diff
@@ -0,0 +1,48 @@
+//
+//  superweatherUITests.m
+//  superweatherUITests
+//
+//  Created by mikael petersson on 2016-04-17.
+//  Copyright © 2016 mikael petersson. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "LocationTool.h"
+
+@interface superweatherUITests : XCTestCase
+
+@end
+
+@implementation superweatherUITests {
+    NSString *latString;
+    NSString *lonString;
+}
+
+- (void)setUp {
+    [super setUp];
+    self.continueAfterFailure = NO;
+    [[[XCUIApplication alloc] init] launch];
+    LocationTool *locationTool = [[LocationTool alloc] init];
+    latString = [NSString stringWithFormat:@"%.1f degrees", [locationTool position].coordinate.latitude];
+    lonString = [NSString stringWithFormat:@"%.1f degrees", [locationTool position].coordinate.longitude];
```

Comment:
What is the expected behaviour if `locationTool == nil` or `[locationTool position]` returns `nil`?